TrafficSimulator/Assets/AutoDrive/AutoDrive.cs

```diff
@@ -320,6 +320,9 @@ private void AddSpanNodes(HashSet<LaneNode> spanNodes)
 
             float nodeDistance = _agent.Context.CurrentNode.DistanceToPrevNode;
 
+            // Occupy nodes further ahead in intersections
+            float intersectionOccupancyOffset = _agent.Context.CurrentNode.Intersection != null ? Intersection.IntersectionLength / 3 : 0;
```

Comment:
Why divide by 3?

Reply:
Explained in [097b264](https://github.com/marc7s/TrafficSimulator/pull/121/commits/097b264811667d464b69394e1c3f23437bea476f)